code/stable/swhs/SRS/HTML/SWHS_SRS.html

```diff
@@ -767,7 +767,7 @@ <h1>Introduction</h1>
           <div class="subsection">
             <h2>Purpose of Document</h2>
             <p class="paragraph">
-              The primary purpose of this document is to record the requirements of the Solar Water Heating System. Goals, assumptions, theoretical models, definitions, and other model derivation information are specified, allowing the reader to fully understand and verify the purpose and scientific basis of SWHS. With the exception of <a href=#Sec:SysConstraints>system constraints</a>, this SRS will remain abstract, describing what problem is being solved, but not how to solve it.
+              The primary purpose of this document is to record the requirements of SWHS. Goals, assumptions, theoretical models, definitions, and other model derivation information are specified, allowing the reader to fully understand and verify the purpose and scientific basis of SWHS. With the exception of <a href=#Sec:SysConstraints>system constraints</a>, this SRS will remain abstract, describing what problem is being solved, but not how to solve it.
```

Comment:
I thought the plan was to have a different name for the NoPCM version of SWHS?  Wasn't the short form going to be SWHS_NoPCM?

Reply:
or SWHSNoPCM, if the underscore is problematic.

Reply:
I believe you are taking a look at the SWHS file, the SWHSNoPCM file does show the correct name. 
![image](https://github.com/JacquesCarette/Drasil/assets/58832908/b99ee7c3-3911-4a8a-aa24-2c3b5c850df6)

Reply:
On second look, I was looking at the wrong file.  I thought it was the NoPCM version, but I see now that it is SWHS.  No problem here.